editor/editor_help.cpp

```diff
@@ -603,8 +603,11 @@ void EditorHelp::_update_doc() {
 	Vector<DocData::MethodDoc> methods;
 
 	for (int i = 0; i < cd.methods.size(); i++) {
-		if (skip_methods.has(cd.methods[i].name))
-			continue;
+		if (skip_methods.has(cd.methods[i].name)) {
+			if (cd.methods[i].arguments.size() == 0 /* getter */ || (cd.methods[i].arguments.size() == 1 && cd.methods[i].return_type == "void" /* setter */)) {
```

Comment:
I think it would be better to prevent adding those to `skip_methods` in the first place, instead of filtering them out on update.

Reply:
Do you recommend that because it saves space?

Reply:
Do you recommend that because it saves space?

Reply:
Well, mainly because it would be cleaner from an implementation point of view. If you parse data to create a map of methods that should be skipped, and you already know at this point that some of those actually shouldn't be skipped, they should not be added to the map in the first place.

That being said, I had a quick look and the `skip_methods` is generated while parsing properties and not methods, so doing the check done here might not be trivial, so I guess the solution you used is the good one.